assets/js/modules/idea-hub/components/setup/SetupForm.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Idea Hub Setup Form component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
+import Checkbox from '../../../../components/Checkbox';
+import Button from '../../../../components/Button';
+import SettingsNotice from '../../../../components/SettingsNotice';
+const { useSelect, useDispatch } = Data;
+
+export default function SetupForm() {
+	const { description } = useSelect( ( select ) => select( CORE_MODULES ).getModule( 'idea-hub' ) );
+	const tosAccepted = useSelect( ( select ) => select( STORE_NAME ).getTosAccepted() );
+	const { setTosAccepted } = useDispatch( STORE_NAME );
+
+	const onChange = useCallback( ( event ) => {
+		const { checked: accept } = event.target;
+		setTosAccepted( accept );
```

Comment:
Why the renaming of `checked` to `accept`? I think
```suggestion
		const { checked } = event.target;
		setTosAccepted( checked );
```
is just as, if not more readable/scan-able. `isChecked` would be better, but not worth the renaming 😉

Reply:
I just followed an example from elsewhere with a checkbox. No worries, I've removed it 👍